dist/rzslider.js

```diff
@@ -764,7 +764,7 @@
             tick.tooltipPlacement = this.options.vertical ? 'right' : 'top';
           }
           if (this.options.showTicksValues) {
-            tick.value = this.getDisplayValue(value, 'tick-value');
+            tick.value = this.getDisplayValue(value, 'rz-tick-value');
```

Comment:
This line should not be modified in my opinion. It's not about css class but it is a simple key that is used for the translate option to determine which value we are translating.